_posts/2001-01-03-customize-project.md

```diff
@@ -130,6 +130,27 @@ The access restriction feature is intended for use with special roles that can b
 
 ------------
 
+## Webhooks
+
+<figure>
+<img alt="Project Webhooks Settings tab" src="{{ site.url }}/images/341-project-webhooks-settings.png" />
+<figcaption>Project Webhooks Settings tab</figcaption>
+</figure>
+
+Webhooks feature are HTTP callbacks which triggered when a document in a language has reached a certain milestone. Currently, webhooks event will be trigger when
```

Comment:
**The** Webhooks feature **is** HTTP callbacks which **are** triggered when a document in a language has reached a certain milestone. Currently, webhooks event**s** will be trigger**ed** when